src/main/java/com/facebook/openwifirrm/optimizers/channel/RandomChannelInitializer.java

```diff
@@ -37,13 +38,39 @@ public class RandomChannelInitializer extends ChannelOptimizer {
 	public static final String ALGORITHM_ID = "random";
 
 	/** The PRNG instance. */
-	private final Random rng = new Random();
+	private final Random rng;
+
+	/** Whether to set a different value per AP or use a single value for all APs */
+	private final boolean setDifferentChannelsPerAp;
 
 	/** Constructor. */
 	public RandomChannelInitializer(
 		DataModel model, String zone, DeviceDataManager deviceDataManager
+	) {
+		this(model, zone, deviceDataManager, false, new Random());
```

Comment:
nit: drop the last argument here (`new Random()`) as you can just keep chaining down, which is cleaner code IMO

Reply:
Good catch, agreed